### PR TITLE
fix(ux): changes the styles of tags

### PR DESCRIPTION
**Pull request by arunkumars08**
fix(ux): changes the styles of tags

As per UX, the styles of tags have to be changed. This is how it will look once this PR will get merged.
screen shot 2018-04-11 at 8 19 27 pm

fix(bug): handles the pipeline summary view bug
In Pipeline card of the summary tab, the pipelines look distorted. This happens if you select a pipeline with more number of stages.
screen shot 2018-04-11 at 6 30 37 pm

And this PR handles that and after mer
screen shot 2018-04-11 at 8 22 38 pm
ging, the card will look like below

### CODE REVIEW

**Comment by arunkumars08** [comment]
@mindreeper2420 I have currently handled for Redhat Suggests tag in HTML as only that comes from the API.

For the rest I have created Generic classes, so once the values start appearing from the backend, we can handle that in HTML by using those classes. 

UX used as reference - https://redhat.invisionapp.com/share/VGF1RF15J#/screens/286890970

Please review.

File: src/app/launcher/create-app/project-summary-createapp-step/project-summary-createapp-step.component.html [hunk 1]
```diff
@@ -116,14 +116,37 @@ <h2 class="card-pf-title">
...
+                    </div>
+                  </div>
+                  <div class="f8launcher-tags" [ngClass]="{'suggested-feature-tag': launcherComponent.summary?.pipeline?.suggested === true}"
+                        *ngIf="launcherComponent.summary?.pipeline?.suggested === true || launcherComponent.summary?.pipeline?.techpreview === true">
                     <span class="f8launcher-tags-label suggested" container="body" triggers="click"
                           outsideClick="true"
                           popover="This pipeline provides an end-to-end process that moves your application from source code to production, with stages to build and test new versions, rollout to staging, review changes, await approval, and promote to production."
                           *ngIf="launcherComponent.summary?.pipeline?.suggested === true">
-                      Suggested <i class="pficon pficon-info"></i>
+                      Redhat Suggests <i class="pficon pficon-info"></i>
```
Comment (mindreeper2420) [inline src/app/launcher/create-app/project-summary-createapp-step/project-summary-createapp-step.component.html]: change to "Red Hat Suggests"

File: src/app/launcher/create-app/release-strategy-createapp-step/release-strategy-createapp-step.component.html [hunk 2]
```diff
@@ -24,52 +24,55 @@ <h2 class="card-pf-title">
...
+                            </div>
+                          </div>
+                        </div>
+                        <div class="f8launcher-tags" [ngClass]="{'suggested-feature-tag': pipeline.suggested === true}"
+                              *ngIf="pipeline.suggested === true || pipeline.techpreview === true">
+                            <span class="f8launcher-tags-label suggested" container="body" triggers="click"
+                                  outsideClick="true"
+                                  popover="This pipeline provides an end-to-end process that moves your application from source code to production, with stages to build and test new versions, rollout to staging, review changes, await approval, and promote to production."
+                                  *ngIf="pipeline.suggested === true">
+                              Redhat Suggests <i class="pficon pficon-info tag-icon"></i>
```
Comment (mindreeper2420) [inline src/app/launcher/create-app/release-strategy-createapp-step/release-strategy-createapp-step.component.html]: change to "Red Hat Suggests"

File: src/assets/stylesheets/shared/_labels.less [hunk 3]
```diff
@@ -1,38 +1,70 @@
...
+        height: 3px;
+        position: absolute;
+        top: 0;
+        right: 0;
     }
-    /* labels are ordered */
-    &.prerequisite {
-      background-color: @color-pf-orange;
+    &.advanced-feature-tag {
+        @advanced-color: #ec7908;
```
Comment (mindreeper2420) [inline src/assets/stylesheets/shared/_labels.less]: can you use a PatternFly color variable instead of a hex code?

**Comment by arunkumars08** [comment]
@mindreeper2420 Addressed all the changes, please have a look

**Pull request merged**